src/cryptoadvance/specter/templates/wallet/history/components/total_wallet_balances.jinja

```diff
@@ -9,23 +9,24 @@
     <h1>
         <small style="line-height:30px">Total balance:</small><br>
         <span style="color: #fff">
-            <span id="fullbalance_amount">{{ wallet.fullbalance | btcunitamount }}</span>
+            <span id="fullbalance_amount">{{ wallet.full_available_balance | btcunitamount }}</span>
             {% if specter.unit == 'sat' %}
 				sats
 			{% else %}
 				{% if specter.is_testnet %}t{%endif%}{% if specter.is_liquid %}L{%endif%}BTC
 			{% endif %}
         </span><br>
         <span class="note">{{ wallet.fullbalance | altunit }}</span>
-        {% if wallet.balance.get("untrusted_pending", 0) or wallet.balance.get("immature", 0) %}<br>
-            <small>( {{ wallet.balance.get("trusted", 0) | btcunitamount }} confirmed,
-                     {% if wallet.balance.get("untrusted_pending", 0) %}
-                        {{ wallet.balance.get("untrusted_pending", 0) | btcunitamount }} pending
-                     {% endif %}
-                     {% if wallet.balance.get("immature", 0) %}
-                        {{ wallet.balance.get("immature", 0) | btcunitamount }} immature
-                     {% endif %}
-            )</small>
+        {% if wallet.balance.get("untrusted_pending", 0) or wallet.balance.get("immature", 0) %}
+        <div class="smaller-card">
```

Comment:
![image](https://user-images.githubusercontent.com/60378539/157185870-e995e0d9-10c3-4197-837c-4a09a5af081b.png)
It would be even nicer if the amounts would be _left_ aligned, so the decimal separator is at the same x position.   And for completeness the unit  (tBTC) after each amount would be good too.

Reply:
Good points, I've created a separate issue for that since this was not really meant to be in this PR in the first place.